src/krate.rs

```diff
@@ -505,7 +505,7 @@ impl Crate {
             downloads,
             description,
             homepage,
-            documentation,
+            mut documentation,
```

Comment:
You can remove this `mut` and just do `let documentation =` below. We're never actually mutating anything.